scene/gui/control.cpp

```diff
@@ -1739,12 +1739,17 @@ void Control::_size_changed() {
 	data.size_cache = new_size_cache;
 
 	if (is_inside_tree()) {
+		// Ensure global transform is marked as dirty before `NOTIFICATION_RESIZED` / `item_rect_changed` signal
+		// so an up to date global transform could be obtained when handling these.
+		if (pos_changed || size_changed) {
+			_notify_transform();
+		}
+
 		if (size_changed) {
 			notification(NOTIFICATION_RESIZED);
 		}
 		if (pos_changed || size_changed) {
```

Comment:
Suggestion for reducing a bit of code:
```suggestion

			if (size_changed) {
				notification(NOTIFICATION_RESIZED);
			}
```

Reply:
Done.